Document support for 2D per-instance uniforms in Shading language (#10749)
godotengine/godot-docs: tutorials/shaders/shader_reference/shading_language.rst

.. note::

>     Since Godot 4.4, per-instance uniforms are available in ``canvas_item`` (2D)
>     and ``spatial`` (3D) shaders. Prior to Godot 4.4, per-instance uniforms are
>     only available in ``spatial`` shaders.

Reviewer: ```suggestion
    Since Godot 4.4, per-instance uniforms are available in both ``canvas_item`` (2D)
    and ``spatial`` (3D) shaders.
```
I don't think we should explicitly mention older versions like this?
